recipes/r-mmcpcounter/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.1.0" %}
+
+package:
+  name: r-mmcpcounter
+  version: {{ version }}
```

Comment:
The mcp counter stuff shoudn't be here, right?

Reply:
Hmmm I have no idea why it appears
Those changes should have already been merged

Reply:
In the last commit I did there were just the files concerning the last package, ConsensusTME, while mMCPcounter was already good

Reply:
I think because this repo uses the "Squash and Merge" strategy, github cannot figure out that the commit is already in the main branch. In this case, all you can do is to add another commit which deletes those filese from this PR.